ts/api/wallet/wallet-api.ts

```diff
@@ -250,6 +250,10 @@ export class WalletAPI {
     return transactions[id];
   }
 
+  public static getCard(index: number): Readonly<CreditCard> {
```

Comment:
there's exactly the same method in [this other PR](https://github.com/teamdigitale/italia-app/pull/242/files#diff-0dc7693a5ad51838135276b8279fbf10R273) - which PR is responsible for providing this method?

Reply:
maybe I can just delete the method being replaced with selectors in next days